<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Pami Automation</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../Downloads/cropped-cropped-iso-bioimagenes2-32x32-1-32x32.png</normaloff>../../../Downloads/cropped-cropped-iso-bioimagenes2-32x32-1-32x32.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">QWidget#centralwidget {
    background-color: #0159A9;
}</string>
   </property>
   <widget class="QWidget" name="loginWidget" native="true">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>290</width>
      <height>380</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget#loginWidget {
    background-color: #fff;
    border-radius: 10px;
    border: 1px solid rgba(1, 89, 169, 0.2);
}</string>
    </property>
    <widget class="QToolButton" name="startAutomation">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>240</y>
       <width>190</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
    background-color: #0159A9;
    color: white;
    border: none;
    border-radius: 8px;
    font-size: 14px;
    font-weight: bold;
}
QToolButton:hover {
    background-color: #003d7a;
}
QToolButton:pressed {
    background-color: #002a5c;
}</string>
     </property>
     <property name="text">
      <string>Iniciar Automatizacion</string>
     </property>
    </widget>
    <widget class="QLabel" name="backgrounImage">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>15</y>
       <width>270</width>
       <height>66</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../Downloads/Bioimagenes2.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="userInput">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>120</y>
       <width>190</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: 2px solid #e0e0e0;
    border-radius: 5px;
    padding: 8px;
    font-size: 14px;
}
QLineEdit:focus {
    border-color: #0159A9;
}</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="passInput">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>185</y>
       <width>190</width>
       <height>35</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
    border: 2px solid #e0e0e0;
    border-radius: 5px;
    padding: 8px;
    font-size: 14px;
}
QLineEdit:focus {
    border-color: #0159A9;
}</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
     <property name="placeholderText">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="userLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>100</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: #333;
    font-size: 12px;
    font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>Usuario</string>
     </property>
    </widget>
    <widget class="QLabel" name="passLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>165</y>
       <width>80</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel {
    color: #333;
    font-size: 12px;
    font-weight: bold;
}</string>
     </property>
     <property name="text">
      <string>Contrase√±a</string>
     </property>
    </widget>
    <widget class="QToolButton" name="logsButton">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>300</y>
       <width>190</width>
       <height>40</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QToolButton {
    background-color: #0159A9;
    color: white;
    border: none;
    border-radius: 8px;
    font-size: 14px;
    font-weight: bold;
}
QToolButton:hover {
    background-color: #003d7a;
}
QToolButton:pressed {
    background-color: #002a5c;
}</string>
     </property>
     <property name="text">
      <string>Logs</string>
     </property>
    </widget>
    <zorder>backgrounImage</zorder>
    <zorder>startAutomation</zorder>
    <zorder>userInput</zorder>
    <zorder>passInput</zorder>
    <zorder>userLabel</zorder>
    <zorder>passLabel</zorder>
    <zorder>logsButton</zorder>
   </widget>
   <widget class="QPushButton" name="settingsButton">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Configuracion</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
